### PR TITLE
Resolve #13 -- Added skin select 

### LINKED ISSUE #13: Add feature to choose skin
Title

### PR DESCRIPTION
**Pull request by Heymeelie**
Added skin select
More adapted for smaller screens
Some fancy button

Refs #13

### CODE REVIEW

**Comment by MatthewFrench** [comment]
Looks good to me.

**Review by MatthewFrench — APPROVED** [review]
Looks good.

File: Client/Javascript/MainPage.js [hunk 1]
```diff
@@ -222,4 +263,4323 @@ var ChampionList = [
     "Ziggs",
     "Zilean",
     "Zyra"
-]
\ No newline at end of file
+]
+
+var extendedChampData = {
```
Comment (MythicManiac) [inline Client/Javascript/MainPage.js]: Could this be moved to a json file or something the like? out of this file anywho.

**Review by MythicManiac — CHANGES_REQUESTED** [review]
When you make changes, add the "Refs #13" to the commit message as well (currently found only on PR message)

No other complaints

**Review by MythicManiac — APPROVED** [review]

**Pull request merged**